### PR TITLE
Portfolio contact form and finish mobile version.

### PR DESCRIPTION
**Pull request by maximoortelli**
_Portfolio contact form and finish mobile version is an exercise where I will add a contact form to my portfolio.The goal here is to master all of the tools and best practices learned about in previous steps to use them in all Microverse projects and most likely in my future job as well._

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,1917 @@
...
+    margin-bottom: 17.708vw;
+  }
+
+  .tarjetas {
+    margin: 6.861vw;
+  }
+
+  .trabajo {
+    border-radius: 1.667vw;
+  }
```
Comment (franclobo) [inline styles.css]: - [x] Kudos for the UI, you almost approved. I suggest adding the hover animations to the cards as you can see in the Figma design to be similar to it, for more details check the image below it has a blue border and a shadow below the card. 👇🏽 

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @maximoortelli, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️
- Correct Gihub flow ✔️
- Good UI ✔️
- Correct use of media queries ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (maximoortelli) [reply styles.css]: Done, thank you franclobo 👍😃

**Review by Munsa1 — APPROVED** [review]
Hi @maximoortelli, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Achieved A Fully Responsive Design :heavy_check_mark: 
Made Requested Changes :heavy_check_mark: 
Neat UI Design :heavy_check_mark: 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**